spec/requests/api/v1/category_data_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'Category Data', type: :request do
+
+  describe 'GET /api/v1/category_data' do
```

Comment:
We have one file in /api/v1/ called restaurant_data.rb, so we keep the same naming.